### PR TITLE
WIP: Show issue with dash

### PR DESCRIPTION
**Pull request by DblK**
This should prevent the issue #148 to happen again once the problem solve.
Since I need a CI to test this out, I am using this PR ;)

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -112,7 +112,7 @@
     homebrew_cask:
       name: "{{ item.name | default(item) }}"
       state: present
-      install_options: "{{ item.install_options | default('appdir=' + homebrew_cask_appdir) }}"
+      #install_options: "{{ item.install_options | default('appdir=' + homebrew_cask_appdir) }}"
```
Comment (geerlingguy) [inline tasks/main.yml:115]: Just wondering why this is being commented out?

File: tests/test.yml [hunk 2]
```diff
@@ -6,7 +6,8 @@
       - ssh-copy-id  # from homebrew/core
       - nginx-full  # from dengi/nginx
     homebrew_cask_apps:
-      - firefox  # from hombrew/cask
```
Comment (geerlingguy) [inline tests/test.yml:9]: It would probably still be good to test a non-dash package too.

**Comment by geerlingguy** [comment]
Thanks for posting this—see #152 and the comments on #148. This is a bug in Ansible upstream, unfortunately :(

**Comment by DblK** [comment]
Yep, I am fairly new to ansible, so I was checking from where it has the issue.
I found out also about the source afterward.

You should add the dash package later once fixed in upstream ;)

**Pull request closed without merging**